components/lib/crash/src/main/java/mozilla/components/lib/crash/CrashReporter.kt

```diff
@@ -96,13 +97,33 @@ class CrashReporter(
         }
     }
 
+    /**
+     * Submit a crash report to all registered services.
```

Comment:
nit: "to all registered services" -> "to all registered telemetry services"?

I wonder if that needs to be public at all. Are we expecting anyone to call this?

Reply:
I don't think this needs to be public.  Are you thinking that someone would want to submit a caught exception to telemetry services?

Reply:
Actually I take that back.  If I'm moving this to the service then I'll have to make this public.